### DISCUSSION TITLE
Did google kill invidious/freetube?

### DISCUSSION THREAD

**Discussion by MattyGWS** [FreeTubeApp/FreeTube#5468] [General]
As the title says, freetube and also invidious in general seems to no longer function, it started late last night but it has remained broken all day today. :( I really hope there is a fix and it wasn't intentional from google to break it.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Nothing to worry about follow this thread https://github.com/FreeTubeApp/FreeTube/issues/5456
